app/application/services/auth/signin_service.py

```
@@ -0,0 +1,31 @@
+class SignInService:
+    def __init__(
+        self,
+        client: LmsClient,
+        jwt_service: JWTService,
+        auth_repository: AuthRepository,
+    ) -> None:
+        self._client = client
+        self._jwt_service = jwt_service
+        self._auth_repository = auth_repository
+
+    async def signin(
```

Ruff (0.12.2)

Missing return type annotation for public function `signin`

(ANN201)